tasks.py

```diff
@@ -4,18 +4,18 @@
 from pathlib import Path
 from invoke import run, task
 
-from mathlibtools.lib import LeanProject
+# from mathlibtools.lib import LeanProject
```

Comment:
Stuff that only relevant to Lean 3 are commented out for your inspection, they can be removed after confirmation.